docs/rdd-programming-guide.md

```diff
@@ -276,7 +276,7 @@ $ PYSPARK_DRIVER_PYTHON=jupyter PYSPARK_DRIVER_PYTHON_OPTS=notebook ./bin/pyspar
 
 You can customize the `ipython` or `jupyter` commands by setting `PYSPARK_DRIVER_PYTHON_OPTS`.
```

Comment:
BTW, this seems broken in Spark due to how [find-spark-home](https://github.com/apache/spark/blob/master/bin/find-spark-home) works in PySpark, see https://github.com/apache/spark/pull/28256